test/factories/interviews.rb

```diff
@@ -1,9 +1,10 @@
 FactoryGirl.define do
   factory :interview do
-    id 155
+    id 1
     firstname { Faker::Lorem.characters(25) }
     lastname  { Faker::Lorem.characters(50) }
     target_level 'beginner'
+    user_id 1
```

Comment:
and more magic number

```id 1``` id is populated automatically so no need to set it explicitly
``` user_id 1 ``` investigate how to work with relations with FacrotyGirl and rework